src/Backoffice/Task/Domain/Enums/TaskStatus.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Task\Domain\Enums;
+
+enum TaskStatus: string
+{
+    case ToDo = 'toDo';
+    case InProgress = 'inProgress';
+    case Accepted = 'accepted';
```

Comment:
Use same name to avoid future errors

```suggestion
    case ToDo = 'ToDo';
    case InProgress = 'InProgress';
    case Accepted = 'Accepted';
```